### ISSUE TITLE
Component audit

### ISSUE THREAD

**Issue by metonym** [IBM/carbon-components-svelte#344]
An extensive component audit must be performed before entertaining a v1 release.

### Identify non-reactive, exported props

Issues #343 , #234

### Identify inconsistent or confusing prop/value nomenclature

- size values are inconsistent (e.g. "small" vs."sm", `undefined` vs. "default")
- ambiguous prop value (For `PaginationNav`, the `page` prop is an index value while in `Pagination` the same prop represents the current page number)
- "is it slottable?" - make prop names more semantic (e.g. "labelText" must be a string while "label" signifies that it can be slottable)

### Identify feature delta compared to flagship Carbon Design System

The basic expectation is that components from the library are compatible with the latest CSS from `carbon-components`.

What features are missing compared to their counterparts in the [flagship Carbon library](https://github.com/carbon-design-system/carbon)?

**Comment by orta** [comment]
I'd recommend also taking a look at the breakpoints for the UIShell when you have [both a sidebar and navigation items](https://carbon-svelte.vercel.app/framed/UIShell/HeaderNav), as at larger width it shows both

**Comment by orta** [comment]
( not trying to pile on, and I'm finding work-arounds by using the DOM API - so it's not a blocker for me, but better to report I assume )

Spotted this also [Dropdown `selectedIndex`](https://codesandbox.io/s/peaceful-http-ok29x?file=/App.svelte) also is not reactive.

**Comment by metonym** [comment]
@orta I'll open separate issues for those

**Comment by ispyinternet** [comment]
Should Toggle and ToggleSmall be a configurable Toggle ?

**Comment by metonym** [comment]
@ispyinternet I agree. Besides the size, `Toggle` and `ToggleSmall` are pretty much identical. Related https://github.com/carbon-design-system/carbon/issues/6653
